src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -586,18 +631,21 @@ internal class DataSource
         private const string DefaultSqlServerInstanceName = "mssqlserver";
         private const string PipeBeginning = @"\\";
         private const string PipeToken = "pipe";
+        private const string LocalDbHost = "(localdb)";
 
         internal enum Protocol { TCP, NP, None, Admin };
 
         internal Protocol ConnectionProtocol = Protocol.None;
 
         internal string ServerName { get; private set; }
+
         internal int Port { get; private set; } = -1;
 
         public string InstanceName { get; internal set; }
 
         public string PipeName { get; internal set; }
```

Comment:
Can we have brief comment about what is different from `PipeName` and `ServerName` for others?